include/constants/event_object_movement.h

```diff
@@ -235,33 +235,33 @@
 #define MOVEMENT_ACTION_ACRO_WHEELIE_MOVE_UP            0x91
 #define MOVEMENT_ACTION_ACRO_WHEELIE_MOVE_LEFT          0x92
 #define MOVEMENT_ACTION_ACRO_WHEELIE_MOVE_RIGHT         0x93
-#define MOVEMENT_ACTION_ACRO_END_WHEELIE_MOVE_DOWN      0x94
-#define MOVEMENT_ACTION_ACRO_END_WHEELIE_MOVE_UP        0x95
-#define MOVEMENT_ACTION_ACRO_END_WHEELIE_MOVE_LEFT      0x96
-#define MOVEMENT_ACTION_ACRO_END_WHEELIE_MOVE_RIGHT     0x97
 
-// 98-A3 are unknown
+#define MOVEMENT_ACTION_SPIN_DOWN  0x94
+#define MOVEMENT_ACTION_SPIN_UP    0x95
+#define MOVEMENT_ACTION_SPIN_LEFT  0x96
+#define MOVEMENT_ACTION_SPIN_RIGHT 0x97
+ 
```

Comment:
The spaces between adjacent groups (`93`/`94`, `97`/`98`, etc) aren't necessary, the groups are clear from the names.